tests/dummy/config/environment.js

```diff
@@ -6,6 +6,9 @@ module.exports = function(environment) {
     environment: environment,
     rootURL: '/',
     locationType: 'auto',
+    moment: {
+      includeLocales: ['ru']
```

Comment:
Was not sure including all locales just for demo purpose was needed, sure will make the change